pheweb/serve/react/js/components/Pheno.js

```diff
@@ -195,6 +267,12 @@ class Pheno extends React.Component {
 	    </div> :
 	<div>loading</div>
 
+	const is_cs = this.state.credibleSets == null ?
```

Comment:
credibleSets is never put into state? Related to this, would be necessary to show the "Traditional" tab by default if autoreporting results are not available. This way we can conveniently use this for all purposes regardless of whether autoreporting has been run or not